editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3916,39 +3925,44 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	animation_hb->add_child(memnew(VSeparator));
 	animation_hb->hide();
 
-	key_loc_button = memnew(Button("loc"));
-	key_loc_button->set_toggle_mode(true);
-	key_loc_button->set_flat(true);
-	key_loc_button->set_pressed(true);
-	key_loc_button->set_focus_mode(FOCUS_NONE);
-	key_loc_button->add_color_override("font_color", Color(1, 0.6, 0.6));
-	key_loc_button->add_color_override("font_color_pressed", Color(0.6, 1, 0.6));
-	key_loc_button->connect("pressed", this, "_popup_callback", varray(ANIM_INSERT_POS));
-	animation_hb->add_child(key_loc_button);
-	key_rot_button = memnew(Button("rot"));
+	const Color key_accent_color = Color::html("84FFB1");
+  const Color key_icon_color = Color(key_accent_color.r * 1.15, key_accent_color.g * 1.15, key_accent_color.b * 1.15);
```

Comment:
Indentation issue. Note that you can apply clang-format locally to ensure that your code matches the expected style: http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#using-clang-format-locally